lib/google_drive/session.rb

```diff
@@ -402,6 +402,8 @@ def upload_from_source(source, title, params = {})
         file_metadata[:mime_type] = params[:convert_mime_type]
       elsif params.fetch(:convert, true) && IMPORTABLE_CONTENT_TYPE_MAP.key?(content_type)
         file_metadata[:mime_type] = IMPORTABLE_CONTENT_TYPE_MAP[content_type]
+      elsif params[:parents]
```

Comment:
This should be a separated if-clause instead of elsif. Otherwise it doesn't work when both :parents and :convert_mime_type parameters are passed.